aws-sagemaker-hosted-scorer/src/main/java/ai/h2o/mojos/deploy/sagemaker/hosted/controller/ModelsApiController.java

```diff
@@ -0,0 +1,108 @@
+package ai.h2o.mojos.deploy.sagemaker.hosted.controller;
+
+import ai.h2o.mojos.deploy.common.rest.api.ModelApi;
+import ai.h2o.mojos.deploy.common.rest.model.Model;
+import ai.h2o.mojos.deploy.common.rest.model.ScoreRequest;
+import ai.h2o.mojos.deploy.common.rest.model.ScoreResponse;
+import ai.h2o.mojos.deploy.common.transform.MojoScorer;
+import ai.h2o.mojos.deploy.common.transform.SampleRequestBuilder;
+import com.google.common.base.Strings;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class ModelsApiController implements ModelApi {
+
+    private static final Logger log = LoggerFactory.getLogger(ModelsApiController.class);
+
+    private final MojoScorer scorer;
+    private final SampleRequestBuilder sampleRequestBuilder;
+    private Boolean isReady;
+
+    private Boolean getReady() {
+        return this.isReady;
+    }
+
+    /**
+     * Simple Api controller. Inherits from {@link ModelApi}, which controls global,
+     * expected request mappings for the rest service.
+     *
+     * Sagemaker Specific: override `getScore` method to point to requestMapping `/invocations`
+     *                     add `ping` method that points to `/ping` for Sagemaker health checks
+     * See Sagemaker Docs here:
+     * https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html
+     * @param scorer
+     * {@link MojoScorer} initialized class containing loaded mojo, and mojo interaction methods
+     * @param sampleRequestBuilder
+     * {@link SampleRequestBuilder} Simple class for generating sample request.
+     */
+    @Autowired
+    public ModelsApiController(MojoScorer scorer, SampleRequestBuilder sampleRequestBuilder) {
+        this.scorer = scorer;
+        log.info("Mojo pipeline successfully loaded ({}).", scorer.getModelId());
+        this.sampleRequestBuilder = sampleRequestBuilder;
+        this.isReady = true;
```

Comment:
This can never be `false` after c-tor, right? And Spring would surely not call it before c-tor is done. So I think the entire variable is not very useful. Personally, I would tend to just remove it.

Reply:
yea... i just borrowed a version of this from tomk code.... will think on this because in production, sagemaker makes periodic checks to `/ping` to verify whether or not the scorer is alive. Would want something that ends up throwing an error if it fails.

Reply:
Having a `/ping` endpoint is ok, but as long as we always return `true` (which is the case, right?), I would remove the extra unused logic. We can put it back once we know how to actually decide between being healthy or not. Does it make sense or am I missing something 🤔? Not a big deal though :+1:.

Reply:
ahhh i see what you mean, if ever we are able to respond from `/ping`, the server is already alive anyways. should just return true